packages/palette/src/svgs/BellIcon.ios.tsx

```diff
@@ -6,7 +6,7 @@ import { Icon, IconProps } from "./Icon"
 /** BellIcon */
 export const BellIcon: React.SFC<IconProps> = props => {
   return (
-    <Icon {...props} viewBox="0 0 18 18">
+    <Icon {...props} width="14px" height="14px" viewBox="0 0 18 18">
```

Comment:
@lilyfromseattle I would expect to see some changes in these files in the `d=` section (where the path is defined).

I don't think we need to update the width/height here-- that can be done from clients who consume this icon.